sourcegraph/sourcegraph
# Add dependencies search button to repository page

Fixes #32045

<img width="1093" alt="Screenshot 2022-03-02 at 11 30 10" src="https://user-images.githubusercontent.com/6417322/156344593-e46e8acb-fa98-481d-a1f2-e75227d36002.png">

## Test plan



* Enable the `dependenciesSearch` experimental setting in site config
* Navigate to any repository page
* You should see a `Dependencies` button
* If you click on the button you should get redirected to the search page with the query `repo:deps(repoName)`
